test/functional/fixtures/api/raw/select-editable-content/pages/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Select editable content</title>
+</head>
+<body>
+<div id="div1" contenteditable="true">div<p id="p1">with</p>contenteditable<p id="p2">attribute</p></div>
+<div id="div2" contenteditable="true">second
+    <div id="div2child">div child</div>
+    div
+</div>
+<div id="div3" contenteditable="true">third div<p id="p3">with</p>contenteditable<p id="p4">attribute</p>div</div>
+<div id="div4" contenteditable="true" style="display: none;">invisible<p id="p5">5</p>div<p id="p6">6</p></div>
+<div id="div5"><p id="p7">7</p>div<p id="p8">8</p></div>
+
+<textarea id="textarea">
+    123
+    456
+</textarea>
+<script>
+    document.getElementById('div1').addEventListener('mouseup', function () {
+        var selection = window.getSelection();
+        var startNode = this.childNodes[1].childNodes[0];
+        var endNode   = this.childNodes[3].childNodes[0];
+
+        if (selection.anchorNode === startNode && selection.anchorOffset === 0 && selection.focusNode === endNode &&
+            selection.focusOffset === endNode.nodeValue.length)
+            throw new Error('Selected editable content in div');
```

Comment:
since we have `Hybrid`use the approach like [here](https://github.com/DevExpress/testcafe/pull/494/files#diff-2ec3651e8a843bb7555095a9464a08e7R8) in all functional tests